src/resources/material.js

```diff
@@ -178,8 +183,18 @@ class MaterialHandler {
         }
     }
 
+    _onTextureUnload(parameterName, materialAsset, textureAsset) {
```

Comment:
mustn't you also handle cubemap unload?

Reply:
ah - yes - I will add _onCubeMapUnload

Reply:
added - NB there is no existing placeholder system for cubemaps - I didn't add one in this PR - but added an issue to investigate whether they are needed: https://github.com/playcanvas/engine/issues/3076